include/sentry.h

```diff
@@ -1345,13 +1370,35 @@ SENTRY_EXPERIMENTAL_API sentry_uuid_t sentry_transaction_finish(
  * well as its reference by passing in the same Transaction as the one passed
  * into this function.
  */
-SENTRY_EXPERIMENTAL_API void sentry_set_span(sentry_value_t transaction);
+SENTRY_EXPERIMENTAL_API void sentry_set_span(sentry_transaction_t *tx);
```

Comment:
TBH, I don’t really know what to do with this one here. In Rust, I have this ugly as a mf type `TransactionOrSpan` for usage with `Scope::get/set_span` :-(

Reply:
scope(s) aren't exposed at all in the public api, hence why it's being omitted from the signature, yes. the existence of scopes just simply do not exist.